gordo_components/client/client.py

```diff
@@ -631,6 +644,37 @@ def predict(
             future = executor.submit(dataset.get_data)
             return await asyncio.wrap_future(future)
 
+    @staticmethod
+    def _adjust_for_offset(dt: datetime, resolution: str, n_intervals: int = 100):
+        """
+        Adjust the given date by multiplying n_intervals by resolution. Such that
+        a data a 12:00:00 with n_intervals=2 and resolution of '10m' (10 minutes)
+        would result in 11:40
+
+        Parameters
+        ----------
+        dt: datetime
+            Initial datetime to adjust
+        resolution: str
+            A string code capable of being parsed by :meth::`pandas.Timedelta`
+        n_intervals: int
+            Number of resolution steps to take earlier than the given date
+
+        Returns
+        -------
+        datetime
+            The new offset datetime object.
```

Comment:
Nitpick: Punctuation at the return, but not at the parameters.

Reply:
Added punctuation to the prior params. :capital_abcd: